vagrant_provisioning/conf/hermes-console-config.json

```diff
@@ -3,7 +3,7 @@
     "title": "hermes vagrant console"
   },
   "dashboard": {
-    "metrics": "",
+    "metrics": "http://PUBLIC_IP:8082",
```

Comment:
Because if someone changes IP in Vagrantfile will not have to do same in X places.